### PR TITLE
Use shapes for the deployment flow

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8746/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8747/

File: jujugui/static/gui/src/app/components/create-payment-user/create-payment-user.js [hunk 1]
```diff
@@ -347,11 +348,17 @@ class CreatePaymentUser extends React.Component {
...
   addNotification: PropTypes.func.isRequired,
-  createCardElement: PropTypes.func.isRequired,
-  createToken: PropTypes.func.isRequired,
-  createUser: PropTypes.func.isRequired,
-  getCountries: PropTypes.func.isRequired,
   onUserCreated: PropTypes.func.isRequired,
+  payment: shapeup.shape({
+    createUser: PropTypes.func.isRequired,
+    getCountries: PropTypes.func.isRequired,
+    reshape: shapeup.reshapeFunc
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/create-payment-user/create-payment-user.js]: I don't see `reshape` being used anywhere. I also don't believe you need to pass it through.

Comment (huwshimi) [reply jujugui/static/gui/src/app/components/create-payment-user/create-payment-user.js]: Thanks, I've cleaned these up and left only the ones that are required by `addReshape`.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8755/

**Comment by huwshimi** [comment]
Thanks @hatched :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**